### DISCUSSION TITLE
How can I use cargo-multivers with winresource?

### DISCUSSION THREAD

**Discussion by xangelix** [ronnychevalier/cargo-multivers#9] [Q&A] [answered]
It appears that cargo-multivers overwrites the windows manifest that I am trying to write with https://crates.io/crates/winresource . Is there a way in which the winresource buildscript could be run _after_ multivers for windows builds?

**Comment by ronnychevalier** [comment, marked as answer]
It is not overwritten per se, it is just not copied from the builds into the final executable.
So currently it is not supported (if you want to do it at the moment, you would have to do it manually by calling `rc.exe` like `winresource` does).

Feel free to open an issue (or a PR :)) to have proper support for this use case. We could either copy the resources from one build into the final executable or provide a custom `build.rs` for the final executable.

**Reply by xangelix** [reply]
I ended up having a lot of issues with `rc` for some reason, so I wrote a bad quick way to overwrite the build.rs in multivers using the winresource crate. I'd love to set up a quick PR, but extremely short on time at the moment. I'll submit an issue for tracking for now.

**Comment by ronnychevalier** [comment]
Since 0.12.0, it is possible to provide a custom runner manifest path with the option `--runner-manifest-path`. Using this, you can use a build script and rely on crates like `winresource`.
